test/ui/tests/fail/wrong-context.stderr

```diff
@@ -0,0 +1,77 @@
+error: Expected `&mut Cx` instead of `Channel`.
+ --> tests/fail/wrong-context.rs:2:22
+  |
+2 | fn sync_channel(_ch: Channel) {}
+  |                      ^^^^^^^
+
+error: Expected `&mut Cx` instead of a `Channel` reference.
```

Comment:
I wonder if we should consider creating Neon compiler error codes similar to rustc and tsc. Just something we could think about for a future PR.

Reply:
Oooh, I like that idea. That would give us something to easily cross-reference to get more details. I'll file that for future improvement.